textproc/chordii/Portfile

```diff
@@ -0,0 +1,52 @@
+PortSystem          1.0
+
+name                chordii
+version             4.5.1
+revision            0
```

Comment:
It seems MacPorts contains at least several hundreds of Portfiles that have this line and `port lint` isn't complaining about it either. Is using an optional keyword really a problem?